tests/Commands/UserModelGeneratorTest.php

```diff
@@ -18,10 +19,17 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        CITestStreamFilter::$buffer = '';
+        // @phpstan-ignore-next-line
```

Comment:
This can be added to phpstan's dynamic constants.

Reply:
Thanks. It is better! Done.